.github/workflows/main.yml

```diff
@@ -0,0 +1,14 @@
+name: CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  test-plugin:
+    name: "Test Plugin"
+    runs-on: macos-latest
```

Comment:
I think we should specify a version so this doesn't break without a code change.